verifiersdemo/source/cpp/src/KaaDemo.cpp

```diff
@@ -66,6 +66,11 @@ int main()
 
     kaaClient->getEventFamilyFactory().getVerifiersDemoEventClassFamily().addEventFamilyListener(messageListener);
 
+    std::cout << "Press Enter to stop" << std::endl;
```

Comment:
Actually, any key stops the application: `cin.get()` waits for any key, though you'd better discuss the messages with other people which also write sample applications based on java sdk, to keep the messages consistent.